src/main/java/com/refinedmods/refinedstorage/apiimpl/autocrafting/task/v6/Processing.java

```diff
@@ -48,6 +48,7 @@
         this.itemsToDisplay = CraftingTask.readItemStackList(tag.getList(NBT_ITEMS_TO_DISPLAY, Constants.NBT.TAG_COMPOUND));
     }
 
+    @Override
```

Comment:
?

Reply:
part of the cleanup for the `finishCalculation` call. Now a bit prettier :)

Reply:
but why is it suddenly `@Override`? I don't see this method in `Craft`?